src/librustdoc/html/static/themes/light.css

```diff
@@ -67,21 +67,11 @@ pre {
 }
 
 .docblock h1, .docblock h2, .docblock h3, .docblock h4, .docblock h5 {
-	border-bottom-color: #DDD;
-}
-
-.docblock table {
-	border-color: #ddd;
-}
-
-.docblock table td {
-	border-top-color: #ddd;
 	border-bottom-color: #ddd;
 }
 
-.docblock table th {
-	border-top-color: #ddd;
-	border-bottom-color: #ddd;
+.docblock table, .docblock table td, .docblock table th {
+	border-color: #000;
```

Comment:
This color in the light theme got changed from `#ddd` to `#000`. Was that intentional?

Reply:
No it wasn't, good catch!